README.md

```diff
@@ -34,6 +34,11 @@ python3 -m src.api
 curl --header "Content-Type: application/json"   --request POST   --data '{"hands":{"S":["D3", "C6", "DT", "D8", "DJ", "D6", "CA", "C3", "S2", "C2", "C4", "S9", "S7"],"W":["DA", "S4", "HT", "C5", "D4", "D7", "S6", "S3", "DK", "CT", "D2", "SK","H8"],"N":["C7", "H6", "H7", "H9", "CJ", "SA", "S8", "SQ", "D5", "S5", "HK", "C8", "HA"],"E":["H2", "H5", "CQ", "D9", "H4", "ST", "HQ", "SJ", "HJ", "DQ", "H3", "C9", "CK"]}}'   http://localhost:5000/api/dds-table/
 ```
 
+### Configuration ###
+
+The server supports configuration using `server.yaml` file. Configuration file is
+loaded from the current working directory. The repository includes `example_server.yaml`.
```

Comment:
Does this mean that the code will not run "out of the box"? If so, I would rename `example_server.yaml` to `server.yaml`.

Reply:
If no server.yaml is found then it uses the default configuration.

Reply:
I see now! I didn't notice that this was only a warning:

```
except Exception as err:
    app.logger.warning("Unable to load configuration {0}: {1}"
                       .format(config_filename, err))
```

I suggest making the message something like this:

`"Unable to load configuration {0}: {1}. Using defaults."`

Also, I think an f-string would be easier to read. Might as well use them everywhere since we now require a version of Python that supports them.